tests/resources/service_templates/node-cellar/types/aria.yaml

```diff
@@ -0,0 +1,93 @@
+
+policy_types:
```

Comment:
lol I understand, but the problem is this yaml import should be part of the extension code and not part of the tests. I realize it's not there right now but I think you might as well have thrown it over there instead right now, no?
Unless I'm missing something, and you think this duplication will remain even once we do have it as part of the extension..?